src/mcall/mcall.cpp

```diff
@@ -176,6 +177,10 @@ class Opts //Options
 	int reportStrand;		//output *_strand.bed? //this file has been incorporated in *bam.bed file and is only for debuging purpose now.
 	int reportSkip; 		//output *_skip.bed?
 	int reportCombined; 	//output *.bed?
+	int statSplit;   //report is splited into 3 parts: chrom1(human), chrom2(lambda), chrom3(virus)
```

Comment:
Not necessarily 3-partition. 3-partition is our use case, but as a program interface, it can be any partitions. And the default is no partition, i.e., all chromosomes as a whole.